core/templates/dev/head/base.html

```diff
@@ -350,6 +350,8 @@
       {{ include_js_file('expressions/evaluator.js') }}
       {{ include_js_file('expressions/parser.js') }}
 
+      {{ include_js_file('domain/utilities/UrlInterpolationService.js') }}
```

Comment:
FYI: this is the sort of thing @gvishal will be working against (putting services in shared pages), since it unnecessarily bloats pages which don't use them.